src/ui/header/navigation.tsx

```
@@ -0,0 +1,173 @@
+                      role="menu"
+                    >
+                      {item.categories?.map((category, ix) => (
+                        <li
+                          key={`${category.title}-${key}-${ix}`}
+                          className="overflow-hidden flex-grow"
+                          role="presentation"
+                        >
+                          <span 
+                            className="font-medium text-base text-foreground mb-2"
+                            role="menuitem"
+                          >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "menuitem" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "menuitem".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)